assets/services.json

```diff
@@ -13,81 +13,297 @@
+        "||amplifyapp.com^",
+        "||amplifyframework.com^",
+        "||amzn.asia^",
+        "||amzn.com^",
+        "||amzn.to^",
+        "||amznl.com^",
+        "||asfiovnxocqpcry.com.cn^",
+        "||assoc-amazon.cn^",
+        "||associates-amazon.com^",
+        "||audible.com^",
+        "||aws-border.cn^",
+        "||aws-icp-domain-manager.cn^",
+        "||aws-iot-hackathon.com^",
+        "||aws^",
+        "||awsapps.cn^",
+        "||awsapps.com.cn^",
+        "||awsautopilot.com^",
+        "||awsautoscaling.com^",
+        "||awsbraket.com^",
+        "||awscommandlineinterface.com^",
+        "||awsedstart.com^",
+        "||awseducate.com^",
+        "||awseducate.net^",
+        "||awseducate.org^",
+        "||awsglobalaccelerator.com^",
+        "||awsloft-johannesburg.com^",
+        "||awsloft-stockholm.com^",
+        "||awssecworkshops.com^",
+        "||awsstatic.cn^",
+        "||awsstatic.com^",
+        "||awsthinkbox.com^",
+        "||awstrack.me^",
+        "||awstrust.com^",
+        "||boxofficemojo.com^",
+        "||cdkworkshop.com^",
+        "||cloudfront-cn.net^",
+        "||cloudfront-test.cn^",
+        "||cloudfront.cn^",
+        "||cloudfront.net^",
+        "||containersonaws.com^",
         "||createspace.com^",
-        "||aws"
+        "||elasticbeanstalk.com^",
+        "||gameon-masters.com^",
+        "||gdansk-amazon.com^",
+        "||images-amazon.com^",
+        "||imdb.com^",
+        "||imdb.to^",
+        "||imdb^",
+        "||kindle.cn^",
+        "||kindle.co.jp^",
+        "||kindle.co.uk^",
+        "||kindle.com^",
+        "||kindle.de^",
+        "||kindle.es^",
+        "||kindle.fr^",
+        "||kindle.in^",
+        "||kindle.it^",
+        "||kindle.jp^",
+        "||kindle^",
+        "||kindleoasis.cn^",
+        "||kindleoasis.com.cn^",
+        "||kindleoasis.com^",
+        "||kindleoasis.info^",
+        "||kindleoasis.jp^",
+        "||kindleoasis.org^",
+        "||kindleoasis.us^",
+        "||kindleoasisnews.com^",
+        "||kindleproject.com^",
+        "||media-amazon.com^",
+        "||media-imdb.com^",
+        "||nwcdcloud.cn^",
+        "||nwcdcloud.com.cn^",
+        "||nwcddns.cn^",
+        "||nwcdinfosec.cn^",
+        "||prime-video.com^",
+        "||primeday.cn^",
+        "||primeday.com.cn^",
+        "||primeday.info^",
+        "||primevideo.cc^",
+        "||primevideo.com^",
+        "||primevideo.info^",
+        "||primevideo.org^",
+        "||primevideo.tv^",
+        "||route53.cn^",
+        "||sagemaker.com.cn^",
+        "||serving-sys.com^",
+        "||siege-amazon.com^",
+        "||ss2.us^",
+        "||ssl-images-amazon.com^",
+        "||thinkboxsoftware.com^",
+        "||ueberamazon.de^",
+        "||xn--cckwcxetd^",
+        "||xn--jlq480n2rg^",
+        "||yamaxun.cn^",
+        "||yamaxun.com^",
+        "||yamaxun^",
+        "||z.cn^",
+        "||zappos^",
+        "/.+\\.awsdns-[0-9][0-9]\\.co\\.uk$/",
```

Comment:
Can this be replaced with `||awsdns-*.co.uk^`?

Reply:
Of course.